src/main/java/sales/Sale.java

```diff
@@ -7,9 +7,8 @@
  * including the quantity sold, the unit price, and the total revenue generated.
  */
 public class Sale {
-    private int quantitySold;
-    private double unitPrice;
-    private double totalRevenue;
+    private Integer quantitySold;
+    private Double unitPrice;
```

Comment:
Great job changing the implementation of the Sale class! We can now assign `null` values to perform checks to display more details to the user!